.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,18 @@
-**Description of work**
+**Description of work.**
+
+**Report to:** [user name]/[nobody]. <!--If the original issue was raised by a user they should be named here.-->
 
 **To test:**
 
 <!-- Instructions for testing. -->
 
 Fixes #xxxx. <!-- and fix #xxxx or close #xxxx xor resolves #xxxx -->
 
-**Release Notes** 
+Does this update require release notes?
+- [ ] Yes
+- [ ] No
 <!--
 Either edit the file in docs/source/release/... and it will be in your pull request or state
```

Comment:
This comment needs updating to make sense now.

Reply:
Done.